docs/topics/http/sessions.txt

```diff
@@ -681,6 +681,61 @@ Technical details
 * Django only sends a cookie if it needs to. If you don't set any session
   data, it won't send a session cookie.
 
+.. _extending-database-backed-session-engines:
+
+Extending database-backed session engines
+-----------------------------------------
+
+.. versionadded:: 1.8
+
+Creating a custom database-backed session engine built upon those included 
+in Django (namely ``db`` and ``cached_db``) may be done via the inheritance
+of the abstract session model :class:`models.AbstractSession`, and an either
+one of ``SessionStore`` classes.
+
+.. class:: models.AbstractSession
+
+    The abstract base session model. Includes everything that is available
+    for the default implementation of the ``Session`` model class.
+
+The example below shows a custom database-backed session engine, that includes
+an additional database column to store an account ID (thus providing an option
+to query the database for all active sessions for an account)::
+
+    from django.contrib.sessions.models import AbstractSession
+    from django.contrib.sessions.backends.db import SessionStore as DBStore
+    from django.db import models
+
+    class CustomSession(AbstractSession):
+        account_id = models.IntegerField(null=True, db_index=True)
+
+        @classmethod
+        def get_session_store_class(cls):
+            return SessionStore
+
+    class SessionStore(DBStore):
```

Comment:
There is a reason — it's an API of `django.contrib.sessions`, this naming is mandatory:
```python
class SessionMiddleware(object):
    def __init__(self):
        engine = import_module(settings.SESSION_ENGINE)
        self.SessionStore = engine.SessionStore

    # ...
```